libminifi/src/core/ProcessSession.cpp

```diff
@@ -568,16 +568,16 @@ void ProcessSession::import(std::string source, std::vector<std::shared_ptr<Flow
 }
 
 bool ProcessSession::exportContent(const std::string &destination, const std::string &tmpFile, const std::shared_ptr<core::FlowFile> &flow, bool keepContent) {
-  logger_->log_info("Exporting content of %s to %s", flow->getUUIDStr().c_str(), destination.c_str());
+  logger_->log_debug("Exporting content of %s to %s", flow->getUUIDStr().c_str(), destination.c_str());
 
   ProcessSessionReadCallback cb(tmpFile, destination, logger_);
   read(flow, &cb);
 
-  logger_->log_info("Committing %s", destination.c_str());
+  logger_->log_debug("Committing %s", destination.c_str());
   bool commit_ok = cb.commit();
 
   if (commit_ok) {
-    logger_->log_info("Commit OK.");
+    logger_->log_debug("Commit OK.");
```

Comment:
I'm on the fence about this one. I usually use the mantra as INFO provides informational statements for support staff, debug and trace used by developers for varying levels. In this case I could see a support staff asking, "Commit OK."  Is there an analog to this statement to provide this information? A totally quiet client at info makes it unusable and more importantly unsupportable by multiple levels of support.